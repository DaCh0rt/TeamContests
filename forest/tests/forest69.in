584 592
8298 7617
-9628 6820
-5172 9266
2006 1209
-9469 6716
6976 4121
-4413 -2982
-1237 2636
-6815 2885
8163 9049
9574 -8233
-9676 83
-9570 2766
-6161 6791
-7301 9247
2364 -3298
-8354 4610
-2840 3235
-7503 7501
-6785 6342
6859 623
2121 -8344
-3203 3956
7337 3777
-1000 -4740
-2569 -5389
-5349 -5212
-7645 -9911
2518 -3107
3938 -2861
7223 4393
-6028 9742
-1427 1021
668 1297
-4662 2198
-9583 -875
6281 4226
-8033 -9966
-3448 -1758
-6675 -834
-8280 -9232
2278 8348
5987 3615
-6133 -6821
4196 -1722
-4917 2064
5373 2749
-7328 4755
8753 -4647
-7554 6923
-3958 -3838
-4661 7568
-7100 -8822
8052 2618
2911 4663
-5982 4439
4629 9014
1090 -5122
9793 -7693
-8952 -1963
-6883 -4706
5787 594
-76 5608
7014 7270
6620 -8001
-9203 -656
7319 -3604
-9611 -7674
1537 7423
8389 -1082
4417 9710
9019 -708
-7863 8223
6232 -4958
-6570 8497
-3241 -8499
6143 -4641
-9973 8814
-9786 7278
7475 1383
9838 -8977
-9331 2127
-2757 8122
9918 4719
-8320 379
7261 -3781
-3293 -8573
-8546 -6063
-293 3424
7512 -4900
-6473 -1000
2630 2807
3782 -8729
7445 5951
7615 3578
1717 -5871
-84 -7439
4546 3037
9874 5765
-5992 2866
5991 1155
-6487 -4187
-3666 7064
-4762 -1342
-4358 7102
8894 4188
2661 3477
8692 -4534
-3416 1402
-5924 -2450
5287 -968
6620 8471
7390 -180
-2722 7181
1119 3094
2999 1096
3659 8126
9484 -2985
9302 5370
937 -3766
-1787 3599
7381 802
1735 5582
-4626 -7412
9774 -2247
7109 -2308
-377 8733
-6626 5366
4589 -2312
-6609 8312
8810 4464
-3638 -4679
-9662 -4976
9098 2534
827 124
-2849 3082
9003 -1101
2997 8544
-2818 9812
-8702 5377
-9112 -106
-8689 -306
6447 1673
5148 -2008
-8450 -3591
6863 7175
-3285 -1277
9073 4006
1432 206
-1816 -3541
5951 -3715
-4142 452
-8940 -9792
-8626 1891
-6561 780
2467 2640
4790 -2461
-2220 437
3443 3602
-9706 -9709
8423 -1668
3106 -1813
6948 -7023
9488 2558
1750 8286
-2096 4903
8237 6764
-9197 -3338
-6440 -5127
-3896 -2236
9765 713
-4795 8206
7654 -7376
-5320 6860
8307 -5833
-9076 8233
-72 5633
8788 1329
7977 9577
886 4876
9164 4842
-6816 -8530
25 -4139
1385 2538
7677 746
2716 6131
-4199 9513
-7141 -8105
-3049 -9549
-8521 -4080
-2331 -9546
-6296 -7882
7683 832
-2545 9417
2097 912
1305 -5239
7167 -1889
9704 -484
-8238 2958
-4755 -276
-4328 -7010
-2270 5410
2531 -5399
4236 8480
7467 -6265
5353 -451
-7365 747
-7592 -8422
-2720 261
5537 -2503
-609 4124
-8033 -1956
-3426 -4471
8148 -4147
7078 -7188
-7944 1318
7617 7369
-3792 8554
-2800 4224
556 2823
2513 -4763
-3187 -2577
9245 4611
-3856 2569
-9996 -2050
-7202 9559
7481 4878
7287 9492
-765 5547
-1752 -9559
-2179 -4644
-9293 1716
-6794 958
2639 5718
2133 6548
5012 -8356
-7067 -6468
2098 -1743
6350 3865
6306 6748
-3631 8867
310 -1530
-7892 -2903
3338 3803
-6882 4969
2408 2633
-2317 -6773
-1674 -6848
-9831 5634
-7711 -3995
-7043 5859
-3857 9406
3155 -1811
-5673 -7410
-9876 -3976
-8025 3853
-7190 7607
2197 2266
-2527 7552
2997 -5988
-3561 2669
-7634 8715
3954 4765
9439 -4177
-3548 -9178
-9830 -8327
-770 766
4395 4630
1991 -1211
3360 -5793
8918 3644
-3106 -777
2070 -8365
-7991 9594
8867 -5968
7134 -5084
-9502 -803
-9785 -8914
3054 -8634
-6461 -5162
9223 -5158
4244 1960
1622 -2604
-285 5917
4057 6015
-3056 -646
7700 -9509
7557 6938
8713 -9782
5662 -7975
-7595 880
-6846 -5914
-8952 9652
-7422 3155
9958 -8645
9534 -5695
-8418 8385
5225 5135
-9546 3556
-8320 3824
1020 9500
4611 8227
-8588 -9006
-91 7916
-126 -1401
2690 -162
7217 -7455
-8852 -961
-5752 8507
-766 -8510
-7115 4863
4285 3595
-9823 4991
9971 -452
-3222 -1296
-1585 -110
-4848 1396
5622 2142
-9856 6623
7092 9798
481 8350
3771 1074
7413 -8768
-2762 6734
-5744 6228
1456 5989
194 -6996
-135 -3517
7438 -2049
404 703
9684 1420
8798 3700
-5591 7015
-4810 -9745
2524 2663
-3768 356
8029 5107
3067 6084
771 7540
-7054 -2171
8158 7824
-744 9574
-9734 -391
7078 4826
9624 -378
-4647 9947
9990 1138
9132 6111
6985 -4451
1022 -9229
-444 -1289
8858 195
3633 7559
485 473
-9130 3206
5197 -1601
6154 8932
-7846 -9981
3235 4987
1185 -3982
-7352 -5323
-5848 -7134
-7106 3069
7424 -5062
297 2881
9138 -9161
-403 -2954
576 6284
-6946 4319
-2364 -7281
-6796 2037
-7376 3626
-5786 -3407
4757 314
7724 5750
-1277 5229
-8068 -2409
-3114 9439
6894 3141
-4421 9209
285 -1353
1541 -8144
-8128 -6026
-580 4090
3177 1930
6989 198
6633 5347
20 1904
-8075 -1646
9279 7609
3929 2836
-8281 -1652
5995 -8286
-2071 -7977
-494 -9711
-3540 -8264
-1410 -8663
1026 -9949
-7235 -6784
-9135 7732
-5079 -4400
-3750 8772
-4409 9238
7185 9992
9507 -6739
1164 1986
-7117 -5882
3478 -1094
2324 -486
-4923 -7566
-9619 -1674
3361 -617
5375 2769
6762 4543
9924 5271
-6048 -6057
8675 -5963
1892 7289
5425 3950
4850 7715
2483 1443
7904 7986
-4141 -4822
-704 -4992
5298 388
-6241 8741
-2791 -1705
4801 3977
8764 1605
8722 -9149
4685 -7856
5210 -8097
745 -3492
7471 -4424
-5599 -9856
-3456 7933
-4482 9241
4742 -6143
-3507 7820
-4705 5675
-3509 1972
-961 -8060
9185 6559
-6653 3067
-7312 1036
-7751 -5170
8063 -6591
-7988 5201
-90 -3231
-3125 7659
-3926 6085
9430 -6627
9685 7751
7977 -9387
8648 3583
-5507 -9028
-4251 9029
9844 5093
-6159 -990
3632 -5272
-831 -6673
9157 -3356
663 -4286
-9382 3999
456 1795
4547 3332
-246 7696
-8506 8914
-5245 545
-9683 -1157
6994 -3662
1637 -5147
9437 2915
9530 -4654
-7271 -8098
-1221 -6360
-2028 9017
-5328 390
-6828 5160
4464 -5546
9423 5153
-8753 1995
-7180 -3577
-8337 -6587
1027 -9752
3848 -1063
-1117 -1370
9880 265
2809 -8063
-8684 6945
5947 6263
-1117 -9553
-6459 -6075
2476 -1886
-6909 -4077
6074 5999
-4124 3303
-2179 -5953
-2591 6060
6749 -6285
7400 7856
5261 8784
6399 -2990
-2460 4092
4216 -7243
-5011 2963
-320 -1603
7457 -62
-8781 7636
2657 3394
8957 2635
-2580 6959
6188 -6293
8184 -8225
-9882 -1431
-6277 -6706
-3852 1445
-3218 1580
-7189 2400
1337 -687
4051 8547
-185 -8796
-869 3593
4346 3271
4691 -9963
-2233 -4510
1069 -3277
-6388 -8820
-9943 5913
6234 -7663
9025 1813
6559 2720
9754 8395
5220 -5759
-4428 -5877
8938 -3074
-1419 100
-9579 4355
-8986 -2834
-9516 4036
5459 -3934
-3122 -4744
7277 -7662
6846 -7164
-192 -9031
6261 7932
-6787 6070
8348 -6131
5182 5612
-9168 6177
-9430 -9903
-1479 -5538
-5672 -7687
-7432 3190
-5546 2677
-4384 -6759
-2235 -6296
-2688 -8393
9936 -5647
8935 9114
-1981 -9429
-8504 9506
5497 9428
5980 -9646
-2346 -4238
-9063 -102
2506 7706
4558 -9388
-1374 -9672
7974 -3084
-2608 -1831
3871 4820
5786 2219
4505 -4376
-8455 3093
6622 -9968
-49 7220
-6675 -2826
-8694 3299
-6430 -9107
-2438 408
-7040 8527
8639 -5516
-7662 8643
