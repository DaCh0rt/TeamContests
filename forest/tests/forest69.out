80038.39 438321621.39
